python/py-gitpython/Portfile

```diff
@@ -16,11 +16,11 @@ long_description    GitPython provides object model access to your git \
                     repository. Once you have created a repository object, you \
                     can traverse it to find parent commit(s), trees, blobs, etc.
 
-checksums           rmd160  26d60c211c58174b885370663af858e2f51a7d33 \
-                    sha256  8e2f57750df91fe72ba5cd1608e4c1954f94bea35c5d735081b2a9b082341bc5 \
-                    size    519437
+checksums           rmd160  8f410b918d5d80ef68398f190b90c6f00d527a86 \
+                    sha256  d0e109a2d9a77927f31a47de5ba658a919d7c0f3110a3cf60e8b6b71a0274fa5 \
+                    size    442510
 
-python.versions     27 36 37
+python.versions     35 36 37
```

Comment:
please add 38 instead of 35. Additionally, you cannot remove the py27-subport here as it still has a depenent:
```
> port echo depends:py27-gitpython
py27-git-sweep
```